### ISSUE TITLE
Incorrect `types` field in `package.json`

### ISSUE THREAD

**Issue by eric-gonzalez-tfs** [spurreiter/cpe-fs#1]
Hi, thanks for publishing this package!

I noticed the `types` field in the `package.json` is incorrect.

https://github.com/spurreiter/cpe-fs/blob/30bac73e9887f87800001c7f1eb5e0daa715e315/package.json#L26

It should specify the file path of the `d.ts` file:
```json
"types": "types/index.d.ts",
```

**Comment by eric-gonzalez-tfs** [comment]
As a temporary solution this worked for me:

```ts
// cpe-fs.d.mts
declare module 'cpe-fs' {
	import * as types from 'cpe-fs/types/index.d.ts';

	export = types;
}
```

**Comment by spurreiter** [comment]
Hi @eric-gonzalez-tfs,
Thanks for rising... A fix with the solution described was published in v1.0.2
